### PR TITLE
feat: live loans powered by flss

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- live loans powered by flss

Local test,
redirect:  https://kiva-ui.local/live-loan/flss/u/5314401/url/1
legacy user img: https://kiva-ui.local/live-loan/flss/u/5314401/img/1
classic user img: https://kiva-ui.local/live-loan/flss/u/5314401/img2/1

### CODE REVIEW

File: server/live-loan-router.js [hunk 1]
```diff
@@ -139,6 +142,27 @@ module.exports = function liveLoanRouter(cache) {
 		});
 	});
 
+	// User URL Router FLSS
+	router.use('/flss/u/:id(\\d{0,})/url/:offset(\\d{0,})', async (req, res) => {
+		await tracer.trace('live-loan.user.redirectToUrl', { resource: req.path }, async () => {
```
Comment (dyersituations) [inline server/live-loan-router.js:147]: I know we aren't actively using the tracers, but is it an issue that this tracer matches the existing `/u` tracing?

Comment (dyersituations) [reply server/live-loan-router.js:147]: Maybe we change to `live-loan.flss`?

Comment (roger-in-kiva) [reply server/live-loan-router.js:147]: Also, https://kiva.atlassian.net/browse/MP-45?focusedCommentId=352345

File: test/unit/specs/server/live-loan-fetch.spec.js [hunk 2]
```diff
@@ -74,6 +74,17 @@ describe('live-loan-fetch', () => {
...
 
+		it('have flss call defined with userId', async () => {
+			fetch.mockClear();
+			fetch.mockResolvedValue({ json: () => {} });
+
+			await fetchLoansByType('user', '1234', true);
+
+			expect(fetch).toBeDefined();
+			const { variables } = JSON.parse(fetch.mock.calls[0][1].body);
+			expect(variables.userId).toEqual(1234);
```
Comment (dyersituations) [inline test/unit/specs/server/live-loan-fetch.spec.js:85]: Can we also ensure that the FLSS method is called (or a query associated with the method)?

File: server/live-loan-router.js (hunk 1 above)
Comment (dyersituations) [reply server/live-loan-router.js:147]: Yup I saw that comment. I just didn't want us to send in data that might confuse existing tracings.

Comment (dyersituations) [reply server/live-loan-router.js:147]: For now I think we can just change these to `live-loan.flss`

**Review by dyersituations — APPROVED** [review]

**Pull request merged**